ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -97,7 +105,7 @@ open class PillButton: UIButton {
 
     private func setupView() {
         setTitle(pillBarItem.title, for: .normal)
-        titleLabel?.font = Constants.font
+        titleLabel?.font = style == .transparent ? Fonts.button1 : Constants.font
```

Comment:
Either move `Fonts.button1` to the constants struct, or move `Constants.Font` out of the Constants struct and call the value directly?